src/App.css

```diff
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+}
+
+.space {
+  width: 100%;
+  margin: 0;
+}
+
+.header {
+  font-family: Georgia;
+  text-align: center;
+  font-size: 30px;
+  color: #fff;
+  height: 120;
+  line-height: 120px;
+  padding-inline: 50;
+  background-color: #af2127;
+}
+
+.content {
+  font-family: sans-serif;
+  font-size: 16;
+  min-height: 120;
+  line-height: 35px;
+  color: #260000;
+  background-color: #fcf5f5;
+  padding-bottom: 5;
+}
+
+.sider {
+  text-align: center;
+  line-height: 90px;
+  color: #000;
+  background-color: #fff0f0;
+  padding-top: 20;
+}
+
+.footer {
+  text-align: center;
+  color: #fff;
+  background-color: #af2127;
+}
+
+p {
+  margin-right: 40;
+  margin-left: 40;
+  margin-top: 25;
+  margin-bottom: 25;
+}
+
+h2 {
+  text-align: center;
+  margin-top: 25;
+}
+
+button {
+  font-family: sans-serif;
+  text-align: center;
+  padding: 10;
+  padding-right: 20;
+  padding-left: 20;
+  margin: 20;
+  border: 0;
+  border-radius: 25px;
+  color: #fff;
+  background-color: #af2127;
+}
```

Comment:
We'll be using antd design system for UI components.
It has the proper margins, paddings and colors. 
```suggestion
```

Reply:
Fixed